setup.py

```diff
@@ -64,12 +64,12 @@
     packages=find_packages(),
     install_requires=[
         'autopep8>=1.5.0,<1.5.6',
-        'click>=7.1.1,<8',
+        'click>=8',
         'colorama',
         'deprecation',
         'entrypoints>=0.3',
         'jinja2>=2.11',
-        'jsonschema>=3.2.0',
+        'jsonschema>=3.2.0,<4.0',  # Cap from KFP
         'jupyter_core>=4.0,<5.0',
```

Comment:
There's no reason to cap the non-existent `jupyter_core` release, could we please remove this?